src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntity_QuantumForceTransformer.java

```diff
@@ -712,8 +712,7 @@ public boolean checkRecipe(final ItemStack aStack) {
 
     private boolean processRecipe(
             ItemStack[] aItemInputs, FluidStack[] aFluidInputs, GT_Recipe.GT_Recipe_Map aRecipeMap, ItemStack aStack) {
-        long tVoltage =
-                getMaxInputVoltage() / getExoticAndNormalEnergyHatchList().size();
+        long tVoltage = getMaxInputVoltage();
```

Comment:
I think that should stay
(EV V + EV V) * (2 A + 2 A) != 4 A EV

Reply:
if you look at how i calculate the amps, i am removing some amps. and afterwards in the OC function remove 0.05% eu/t from the total to get it to use less to account for "cable loss". GDCloud tested it to make sure it worked fine.

Reply:
okay i see what you meant. will try to get a fix.

Reply:
Its definitely gonna take too much power. Look at this example with 2 UV energy hatches
tVolatage = 524288 + 524288 = 1048576
tAmps = (2 + 2) * 0.8 = 3
total = 3145728
WHILE 2 UV hatches can take at most 2097152 EU/t

Reply:
fixed